lambdas/service/app.py

```diff
@@ -1062,7 +1143,7 @@ def fetch_dss_files(uuid):
 
 
 def _dss_files(uuid, fetch=True):
-    query_params = app.current_request.query_params
+    query_params = app.current_request.query_params or {}
```

Comment:
This change isn't specific to the documentation, should it be in a different commit? 
Also this seems like a regression from the present implementation, I could be wrong.

Reply:
It IS in its own commit. Did you mean a different pull request?

Reply:
Yea that was a mistake.

Reply:
Yea that was a mistake on my behalf.